R/fit.R

```diff
@@ -273,6 +273,36 @@ CmdStanMCMC <- R6::R6Class(
       } else {
         cat(paste(self$runset$procs$chain_output(id), collapse="\n"))
       }
+    },
+
+    draws = function(inc_warmup = FALSE) {
+      if (!length(self$output_files())) {
+        stop("No chains finished successfully. Unable to retrieve the fit.",
```

Comment:
I think this needs to go after after checking if `private$draws_` is NULL. It shouldn't error if the temp output files are missing if the user has already read in the csv previously.

Reply:
Actually we don't need to throw an error at all. This error will already be caught by `private$read_csv_()` so we can delete these lines.

Reply:
Good call. Removed.